scripts/ci/run-jenkins.sh

```diff
@@ -6,8 +6,14 @@ export TEST_HARNESS_VERBOSE=1
 
 if [[ ${label} == 'osx-i386' ]]; then EXTRA_CONF_FLAGS="--with-libgdiplus=/Library/Frameworks/Mono.framework/Versions/Current/lib/libgdiplus.dylib --enable-nls=no --build=i386-apple-darwin11.2.0"; fi
 if [[ ${label} == 'osx-amd64' ]]; then EXTRA_CONF_FLAGS="--with-libgdiplus=/Library/Frameworks/Mono.framework/Versions/Current/lib/libgdiplus.dylib --enable-nls=no"; fi
-if [[ ${label} == 'w32' ]]; then PLATFORM=Win32; EXTRA_CONF_FLAGS="--host=i686-pc-mingw32"; export MONO_EXECUTABLE="`cygpath -u ${WORKSPACE}\\\msvc\\\Win32\\\bin\\\Release_SGen\\\mono-sgen.exe`";fi
-if [[ ${label} == 'w64' ]]; then PLATFORM=x64; EXTRA_CONF_FLAGS="--host=i686-pc-mingw32"; export MONO_EXECUTABLE="`cygpath -u ${WORKSPACE}\\\msvc\\\x64\\\bin\\\Release_SGen\\\mono-sgen.exe`"; fi
+if [[ ${label} == 'w32' ]]; then PLATFORM=Win32; EXTRA_CONF_FLAGS="--host=i686-pc-mingw32"; fi
+if [[ ${label} == 'w64' ]]; then PLATFORM=x64; EXTRA_CONF_FLAGS="--host=i686-pc-mingw32"; fi
+
+if [[ ${label} == w* ]]; then
+	export MONO_EXECUTABLE="`cygpath -u ${WORKSPACE}\\\msvc\\\\${PLATFORM}\\\bin\\\Release_SGen\\\mono-sgen.exe`"
+	# without this dir in PATH the P/Invoke tests in mono/tests/ won't find the MSVC built libtest.dll
+	export PATH="$PATH:`cygpath -u ${WORKSPACE}\\\msvc\\\\${PLATFORM}\\\bin\\\Release`"
```

Comment:
Does indeed sound cleaner. But IIUC the make targets that run when `run-jenkins.sh` invokes `test-wrench` never use `tests-config`. Only seems to be used by the `test-driver` perl script which is never invoked. I tried manually adding a 

```
<dllmap dll="libtest" target="C:\Users\Niklas\Desktop\mono\msvc\x64\bin\Release\libtest.dll" />
```

line to `test-config` but it didn't change anything.